src/index.js

```diff
@@ -15,70 +14,44 @@ const {
     userFromRoomDropper,
     messageToRoomSender,
     locationToRoomSender,
-    roomMessagesFetcher,
 } = require('./services')
-const { roomRepo } = require('./repos')
-const { Location } = require('./entities')
+const actions = require('./actions')
+const middlewares = require('./middlewares')
 
 const app = express()
 const server = http.createServer(app)
 const io = socketio(server)
 
-const publicDirPath = path.join(__dirname, '../public')
+app.use(middlewares)
```

Comment:
Is it some sort of a `reserved` word ('middlwares') for app to use? Or does it just use whatever is exported in that file?

Reply:
it is kinda pattern. And it is a part of express 
https://expressjs.com/en/guide/using-middleware.html